### PR TITLE
Uniform hard wrapping

### PR DESCRIPTION
**Pull request by JGiola**
Add hard wrapping for the last two paragraph like the rest of the file

### CODE REVIEW

File: Documentation/DevelopingPackages.md [hunk 1]
```diff
@@ -37,13 +38,17 @@ and you need to work on that package simultaneously
 then you have several options:
 
  1. **Edit the sources the package manager clones**
-    
+
```
Comment (mxcl) [inline Documentation/DevelopingPackages.md]: Doesn’t this break markdown? Now the bullets are separated (I think).

Comment (JGiola) [inline Documentation/DevelopingPackages.md]: I've watched the two previews and I don't see any differences.
By the way those spaces are not there for the point 2 of the same list

**Comment by mxcl** [comment]
I'll merge this if you rebase. I don't want to merge a PR with a merge commit already in it.

**Comment by JGiola** [comment]
I think I've done it.
Sorry for the mess up but its my first time to do multiple PR from the same repository...

**Pull request merged**